src/sentry/scim/endpoints/members.py

```diff
@@ -370,16 +373,29 @@ def post(self, request: Request, organization) -> Response:
         - Sentry's SCIM API doesn't currently support setting users to inactive,
         and the member will be deleted if active is set to `false`.
         - The API also does not support setting secondary emails.
+        - you can provide a sentryOrgRole to provision alongside the new user
         """
 
```

Comment:
should we add this key to the example above? any other docs that need to be updated?

Reply:
I actually dropped this change from the docstring. I don't want this to show up in public docs before it's ready.

Reply:
👍🏼